### DISCUSSION TITLE
Allow scheduling unpublish and then publish again

### DISCUSSION THREAD

**Discussion by NltgRepresentative** [umbraco/Umbraco-CMS#13886] [Features and ideas]
In v7 it was possible to, for a published page, schedule unpublish at date x and publish at date y where x < y. In v9 and later this is not possible anymore and our cms editors are requesting this feature back.

**Comment by bergmania** [comment]
In generel I can say we wanna have more flexibility in our publishing/unpublising, but we have no timeline for when this will be made. 

This would make sense as part of a bigger task, improving and supporting multiple future scheduling versions.
